odoo/orm/fields.py

```diff
@@ -1260,7 +1265,7 @@ def _value_to_column(v):
             can_be_null = False
             if (null_value := self.falsy_value) is not None:
                 value = self.convert_to_cache(value, model) or null_value
-                can_be_null = (
+                can_be_null = nullable and (
```

Comment:
`can_be_null` and `nullable` is quite confusing IMO. Maybe you can rename `can_be_null` (accept_null_value or check_null).

Reply:
Ok for "accept_null_value".